tools/toolchain/prepare

```diff
@@ -43,7 +43,7 @@ for arch in "${archs[@]}"; do
     fi
 done
 
-buildah bud "${archs[@]/#/--platform=linux/}" --jobs 0 --squash --no-cache --pull -f tools/toolchain/Dockerfile --manifest "$(<tools/toolchain/image)"
+buildah bud "${archs[@]/#/--platform=linux/}" --jobs 0 --squash --no-cache --pull -f tools/toolchain/Dockerfile --manifest "$(<tools/toolchain/image)" --build-arg CLANG_BUILD=SKIP
```

Comment:
Can you make this a parameter to prepare too? I use prepare exclusively to prepare images.

Reply:
Yes, of course - it won't work without it. This is just a preparation.

Reply:
Added.